app/utils/RepaymentAmountSummaryAnswersHelper.scala

```diff
@@ -58,13 +58,20 @@ class RepaymentAmountSummaryAnswersHelper(userAnswers: UserAnswers)(implicit mes
         Html(x.map(value => HtmlFormat.escape(message).toString).mkString("")),
         Html(x.map(value => formattedAmount).mkString("")),
         index match {
-          case "0" if isCustomDutyExists => Some(routes.CustomsDutyPaidController.onPageLoad(CheckMode).url)
-          case "1" if isCustomDutyExists => Some(routes.CustomsDutyPaidController.onPageLoad(CheckMode).url)
-          case "0" if isVATExists => Some(routes.VATPaidController.onPageLoad(CheckMode).url)
-          case "1" if isVATExists => Some(routes.VATPaidController.onPageLoad(CheckMode).url)
-          case "0" if isOtherDutiesExists => Some(routes.OtherDutiesPaidController.onPageLoad(CheckMode).url)
-          case "1" if isOtherDutiesExists => Some(routes.OtherDutiesPaidController.onPageLoad(CheckMode).url)
-          case _ => None
```

Comment:
mode == NormalMode can be extracted as an expression and reused. Same for checkMode

Reply:
This is fixed